src/pixelator/analysis/__init__.py

```diff
@@ -26,7 +26,7 @@ class AnalysisParameters:
     compute_polarization: bool
     compute_colocalization: bool
     use_full_bipartite: bool
-    polarization_normalization: PolarizationNormalizationTypes
+    polarization_normalization: PolarizationTransformationTypes
```

Comment:
There is some mixed use of "normalization" and "transformation" here and there. What do you think about that?

Reply:
Should this be `polarization_transformation: PolarizationTransformationTypes`?